server/routes/pubDocument.tsx

```diff
@@ -126,12 +127,19 @@ const getEnrichedPubData = async (options: EnrichedPubOptions) => {
 	};
 };
 
-const speedLimiter = slowDown({
-	windowMs: 60000, // 1 minute for requests to be kept in memory. value of 60000ms is default but expressed here for clarity
-	delayAfter: 60, // allow 60 requests per minute, then...
-	delayMs: 100, // 60th request has a 100ms delay, 7th has a 200ms delay, 8th gets 300ms, etc.
-	maxDelay: 20000, // max time of request delay will be 20secs
-});
+/**
+ * `express-slow-down` causes a memory leak in Jest while testing.
+ * While testing, this is not very useful anyway.
+ */
+const speedLimiter: RequestHandler =
+	process.env.NODE_ENV === 'test'
+		? (req, res, next) => next()
+		: slowDown({
+				windowMs: 60000, // 1 minute for requests to be kept in memory. value of 60000ms is default but expressed here for clarity
+				delayAfter: 60, // allow 60 requests per minute, then...
+				delayMs: 100, // 60th request has a 100ms delay, 7th has a 200ms delay, 8th gets 300ms, etc.
+				maxDelayMs: 20000, // max time of request delay will be 20secs
```

Comment:
Important: I changed maxDelay to maxDelayMs: `maxDelay` is not a property you can set, so did not do anything. This might cause unexpected behavior now it is actually set, although i've not noticed anything

Reply:
Awesome, thanks for pointing this out. We'll keep an eye on it!